Spanner/src/SpannerClient.php

```diff
@@ -137,6 +137,20 @@ class SpannerClient
      *     @type bool $returnInt64AsObject If true, 64 bit integers will be
      *           returned as a {@see Google\Cloud\Core\Int64} object for 32 bit
      *           platform compatibility. **Defaults to** false.
+     *     @type array $queryOptions Query optimizer configuration.
+     *     @type string $queryOptions.optimizerVersion An option to control the
+     *           selection of optimizer version. This parameter allows
+     *           all execute queries to use a specific query optimizer version.
+     *           Specifying "latest" as a value instructs Cloud Spanner to use
+     *           the latest supported query optimizer version.
+     *           The order of precedence for setting the optimizer version is:
```

Comment:
Perhaps something like...

"query-level values will take precedence over any global settings. If the `SPANNER_QUERY_OPTIMIZER_VERSION` environment variable is set, it will take second priority. This value is used when neither a query-level value nor the environment variable is set."

Reply:
Done.